rpcs3/Emu/Cell/lv2/sys_cond.cpp

```diff
@@ -57,7 +58,7 @@ error_code sys_cond_destroy(ppu_thread& ppu, u32 cond_id)
 		}
 
 		cond.mutex->cond_count--;
-		lv2_obj::on_id_destroy(cond, cond.shared, cond.key);
+		lv2_obj::on_id_destroy(cond, !!cond.key, cond.key);
```

Comment:
If there is no named constant, at least write `cond.key != 0`

Reply:
Fixed.

Reply:
Hmm thanks.